node/core/av-store/Cargo.toml

```diff
@@ -7,7 +7,8 @@ edition = "2018"
 [dependencies]
 derive_more = "0.99.9"
```

Comment:
could you take a look if `derive_more` can now be removed here and elsewhere?

Reply:
Planing to add `#[deny(unused_extern_crates)]` to all the crates I touched. I think this is a generally good thing to do.